examples/swarm-setup.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	metrics "github.com/ipfs/go-libp2p/p2p/metrics"
+	tu "github.com/ipfs/go-libp2p/testutil"
+	ma "github.com/jbenet/go-multiaddr"
+	swarm "go-libp2p/p2p/net/swarm"
```

Comment:
If you dont want to type this manually, just run `gx-go rewrite` from the `go-libp2p` directory (the one with the package.json in it)